src/sqlfluff/core/default_config.cfg

```diff
@@ -43,6 +43,8 @@ comma_style = trailing
 allow_scalar = True
 single_table_references = consistent
 unquoted_identifiers_policy = all
+semi_colon_new_line = False
```

Comment:
Isn't "semicolon" a single (compound) word? I'm thinking we don't want that underscore there.
```suggestion
semicolon_newline = False
```

Reply:
I think you can also spell it semi-colon, hence the underscore but happy to do one word 👍 
Should I maybe move that config to a rule specific config as well?

Reply:
I would move it, yes. It's unlikely another rule would use something so specific.

Reply:
Flowed these changes through to code and unit tests